defs/events.yaml

```diff
@@ -85,6 +85,13 @@ openstack:
           expr: '(\S+ \d+) \d+:\S+\s+.+apparmor="DENIED".+\s+profile="(\S+neutron\S+)"\s+.+'
           hint: 'apparmor'
 kernel:
+  memory-check:
+    kernlog:
+      oom:
+        datasource: 'var/log/kern.log'
+        start:
+          expr: '(.+ \d+) (\d+:\d+:\d+) .+ (\S+) invoked oom-killer\: .+'
```

Comment:
you can lose the .+ at the end since everything will match up until that point and we don't need anything beyond.

Reply:
ack, done.